sprints/webhooks/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+import requests
+import json
+
+class WebhookEvent(models.Model):
+    name = models.TextField()
```

Comment:
One last comment, as I've just noticed this wile testing the PR - why aren't we simply using `CharField` here? We don't need textarea in the admin panel for this, as it's just a name.

Reply:
@nizarmah, almost - fe4038e.
When updating models, we also need to add/update migrations. Also, it's a good practice to add `max_length` to the [CharField](https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.CharField).

Reply:
Ah yeah, I keep forgetting about migrations 🤦🏼 

Thanks for the note about adding `max_length` for the CharField 👍🏼